test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/UnaryOperatorTest.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import java.util.function.UnaryOperator
+
+import org.junit.Assert._
+import org.junit.Test
+
+class UnaryOperatorTest {
+  import UnaryOperatorTest._
+
+  @Test def identity_String(): Unit = {
+    val unaryOperatorString: UnaryOperator[String] = UnaryOperator.identity()
+    assertEquals("scala", unaryOperatorString.apply("scala"))
+  }
+
+  @Test def identity_Int(): Unit = {
+    val unaryOperatorString: UnaryOperator[Int] = UnaryOperator.identity()
+    assertEquals(1, unaryOperatorString.apply(1))
+  }
+
+  @Test def should_allow_override(): Unit = {
```

Comment:
I don't understand what this is testing. It seems to be simply a poorly named object / function pair?